<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Цветовые модели</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 175, 255);
</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 175, 255);
</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>C:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>M:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>K:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 175, 255);
</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>H:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>L:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>S:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 61, 136);
font: 900 11pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 61, 136);
font: 900 11pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CMYK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 61, 136);
font: 900 11pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>HLS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 155);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(88, 61, 136);</string>
    </property>
    <property name="text">
     <string>Choose color</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>321</width>
      <height>159</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
